src/VisualStudio/Core/Def/Implementation/FindReferences/Entries/AbstractDocumentSpanEntry.cs

```diff
@@ -70,6 +73,25 @@ protected override object GetValueWorker(string keyName)
                 return null;
             }
 
+            private LinePosition GetLinePosition()
```

Comment:
LinePosition doesn't include the document path (L60) so this model only works when the usage is in the same file. Razor has constructs that can include user-code from multiple files in a single generated document.

Reply:
Other than that, this looks roughly like what I'd expect

Reply:
good point! I should make sure it return location same as the entry is pointing to, if not, should figure out what to do..